components/widgets/icons/mattermost_logo.tsx

```diff
@@ -13,19 +13,19 @@ export default function MattermostLogo(props) {
                 x='0px'
                 y='0px'
                 viewBox='0 0 500 500'
-                style={style.background}
+                enableBackground='new 0 0 500 500'
```

Comment:
Shouldn't this be part of the `style` attribute?
```suggestion
                style={{
                    enableBackground: 'new 0 0 500 500',
                }}
```